src/app/jobs/jobs.component.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, OnInit} from '@angular/core';
+
+
+import {JobService} from './jobs.service';
+import { Job } from './types';
+
+
+@Component({
+  selector: 'app-jobs',
+  templateUrl: './jobs.component.html',
+  styleUrls: ['./jobs.component.scss'],
+})
+export class JobsComponent implements OnInit {
+  jobService: JobService | null;
+  jobs: Job[] | null = [];
+
+  isLoading = true;
+  loadError = false;
+  displayedColumns: string[] = ['id', 'type', 'product', 'state', 'details'];
+
+  constructor() {}
+
+  ngOnInit(): void {
+    this.jobService = new JobService();
```

Comment:
this way you won't let angular inject the singleton, plus if the service has dependencies they won't get injected properly. Please add this in the constructor.

Reply:
Good observation, thanks